Documentation/contributing.rst

```diff
@@ -213,6 +213,17 @@ commands, respectively:
 Testsuite
 ~~~~~~~~~
 
+.. Warning::
+
+  Running the testsuite will modify the host environment. If you are using the
+  default VM that might not be an issue, but if you are running bare metal or a
+  different VM the tests might fail or worst case remove possibly important
+  configuration. Specifically, they are modifying tables and flushing chains,
+  kernel configurations change for IP. Also adding and removing networking
```

Comment:
Does `kernel configurations change for IP` mean assigning IP addresses to devices, or did you have other things in mind?

Reply:
Was thinking about `sysctl` but there other things in the setup script
https://github.com/cilium/cilium/blob/master/bpf/init.sh#L74-L78

Reply:
drop this part or reword it?

Reply:
Maybe just reword it to specify for example, networking options via ``sysctl``?